consensus/pow/utils.go

```diff
@@ -0,0 +1,18 @@
+package consensus
+
```

Comment:
move to common folder

Reply:
consensus is a independent component, there is also consensus folder in ETH. so I would prefer to keep it.